### PR TITLE
Correcting some audio-id mixups and updating audio-ids

### PR DESCRIPTION
**Pull request by n1ghty**
There were some audio-ids assigned to the incorrect Tonie IDs.
10000309 -> 10000909 (audio-id already existed with missing information, but was assigned to both tonies)
10001693 -> 10000696
10001889 -> 10001989

I also added some unknown audio-ids.

I did not check for any further duplicates/mixups.

### CODE REVIEW

File: yaml/10001693.yaml
Comment (SciLor) [inline yaml/10001693.yaml:1]: Why did you remove this one?

**Review by SciLor — APPROVED** [review]

**Comment by SciLor** [comment]
Thank you for your corrections

File: yaml/10001693.yaml
Comment (n1ghty) [reply yaml/10001693.yaml:1]: This audio-id actually belongs to 10000696.

**Comment by n1ghty** [comment]
when should confidence be set to 1?

**Comment by SciLor** [comment]
> when should confidence be set to 1?

Confidence is set, when we can approve the audio-id with an automated process. So 0 is okay in this case.

**Comment by n1ghty** [comment]
The merge-conflict of 10000309 was incorrectly solved. You deleted the wrong audio-id (1632487122 which was just added with the rambudo changes instead of 1691659153)

**Comment by SciLor** [comment]
https://github.com/toniebox-reverse-engineering/tonies-json/blob/master/yaml/10000309.yaml

This one is the right way around now?
